### PR TITLE
[fix] html in attr not escaped

### PR DESCRIPTION
**Pull request by Scrum**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Repo settings
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [x] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

### CODE REVIEW

File: src/core/render/tpl.js [hunk 1]
```diff
@@ -91,7 +93,9 @@ export function tree(toc, tpl = '<ul class="app-sub-sidebar">{inner}</ul>') {
 
   let innerHTML = '';
   toc.forEach(node => {
-    innerHTML += `<li><a class="section-link" href="${node.slug}" title="${node.title}">${node.title}</a></li>`;
+    innerHTML += `<li><a class="section-link" href="${
+      node.slug
+    }" title="${escapeHtml(node.title)}">${node.title}</a></li>`;
```
Comment (anikethsaha) [inline src/core/render/tpl.js:98]: Just one optional thought

 How about escaping `node.title` ?

**Review by anikethsaha — CHANGES_REQUESTED** [review]
Lgtm

One optional thought

File: src/core/render/tpl.js (hunk 1 above)
Comment (Scrum) [reply src/core/render/tpl.js:98]: I didn't really understand the meaning of the question. the problem can be seen here https://posthtml.org/#/packages on sidebar '">'

**Comment by sy-records** [comment]
#1404 will fix this problem

File: src/core/render/tpl.js (hunk 1 above)
Comment (sy-records) [reply src/core/render/tpl.js:98]: fixed via #1336

**Review by sy-records — CHANGES_REQUESTED** [review]
Thank you for your contribution.

But there are some problems:
1. You don't need to modify `package-lock.json`
2. The problem with the `A` tag has been fixed. #1336
3. The `title` shouldn't have any tags, so it will be removed at #1404, not escaped. 
`& " '`neither need be escaped.

**Comment by Scrum** [comment]
> Thank you for your contribution.
> 
> But there are some problems:
> 
>     1. You don't need to modify `package-lock.json`
> 
>     2. The problem with the `A` tag has been fixed. #1336
> 
>     3. The `title` shouldn't have any tags, so it will be removed at #1404, not escaped.
>        `& " '`neither need be escaped.

force update PR

**Comment by sy-records** [comment]
Actually, I think you can close this PR  :)

**Comment by Scrum** [comment]
> The problem with the A tag has been fixed. #1336

**Pull request closed without merging**